src/components/cards/CommentCard.tsx

```diff
@@ -62,27 +95,25 @@ export const CommentCard = ({ comment }: { comment: any }) => {
           )}
         </Stack>
 
-        <TapWriteComment content={comment.content} getContent={() => {}} readonly />
+        <TapWriteComment content={comment.details.content as string} getContent={() => {}} readonly />
 
-        {comment.children?.map((item: any) => {
+        {commentAddedResponseSchema.parse(comment.details)?.replies?.map((item: any) => {
           return (
             <Stack direction="column" rowGap={3} key={item.id}>
               <CustomDivider />
               <Stack direction="row" columnGap={2} alignItems={'center'}>
-                <Avatar
-                  alt="user"
-                  src={comment?.iconImageUrl || comment?.avatarImageUrl}
-                  sx={{ width: '20px', height: '20px' }}
-                />
-                <BoldTypography>{item.details?.initiator}</BoldTypography>
+                <Avatar alt="user" src={''} sx={{ width: '20px', height: '20px' }} />
```

Comment:
Is there a reason why src is empty string? Does it default to something else when empty?

Reply:
Yes. As per @aatbip, right now we dont get AvatarUrl from getInterUsers function from the copilot SDK. I thought once the copilot team gets to do this, I will add the urls.